wp-content/themes/core/components/blocks/core_columns/block-editor.pcss

```diff
@@ -0,0 +1,13 @@
+
+/* -----------------------------------------------------------------------------
+ *
+ * Block Editor
+ * Core Columns Block Styles & Overrides
+ *
+ * Styles applicable to this block in the block editor.
+ *
+ * ----------------------------------------------------------------------------- */
+
+.wp-block[data-type^="core/columns"] {}
```

Comment:
I originally had some admin related styles here. Even though there is nothing left, do we want to keep this file as an example for anyone who needs admin styles?

Reply:
Meh. I say delete this by default. No need to carry around and process an empty css partial on every new project.